src/app/complex/complex-results/complex-results.component.ts

```diff
@@ -32,34 +31,26 @@ export class ComplexResultsComponent implements OnInit, AfterViewInit {
   confidenceFilter = 1;
 
   private _toast;
-  private _listPageSize = 15; // This is where we set the size of the pages for list view
-  private _navigatorPageSize = 20; // This is where we set the size of the pages for navigator view
-  private _listCurrentPage: number;
-  private _navigatorCurrentPage: number;
-  private _listLastPageIndex;
-  private _navigatorLastPageIndex;
-
-  constructor(private route: ActivatedRoute, private router: Router,
+  private pageSize = 20; // This is where we set the size of the pages for navigator view
```

Comment:
Yes we actually removed that because it was a lot of complexity and reloading time just to have 5 iteems less in the list view, which is not that important in my opinion

Reply:
That makes sense, but the comment is not updated.